### DISCUSSION TITLE
Using a local package as typst template

### DISCUSSION THREAD

**Discussion by psads-git** [quarto-dev/quarto-cli#10300] [Q&A] [answered]
I have created a `typst` template, which is a local package:

```
#import "@local/myconf:0.1.0": conf

#show: doc => conf(
  title: [My title],
  lang: "pt",
  doc)

```
Is it possible to use it inside a `quarto` document?

Thanks!

**Comment by mcanouil** [comment, marked as answer]
Quarto works with the partials either directly in the YAML front matter or packaged as an extension as documented.
So, if you write the right thing in the right place, I believe it should work.

- https://quarto.org/docs/journals/templates.html#typst-partials

**Comment by psads-git** [comment]
Thanks, @mcanouil, for the directions!
